plugins/search-backend-node/api-report-alpha.md

```diff
@@ -32,20 +32,21 @@ export const searchIndexRegistryExtensionPoint: ExtensionPoint<SearchIndexRegist
 
 // @alpha
 export interface SearchIndexService {
+  build(options: SearchIndexServiceBuildOptions): void;
```

Comment:
Thinking that `init` or `initialize` could be a better name here? Generally, you'd expect `build` to return something, rather than mutate state.

API-wise it can be nice to force correct ordering of things too, since we're making a breaking change here anyway. We could for example do something like this:

```ts
export interface SearchIndexService {
  build(options: SearchIndexServiceBuildOptions): SearchIndex;
}

export interface SearchIndex {
  getDocumentTypes(): Record<string, DocumentTypeInfo>;
  start(): Promise<void>;
  stop(): Promise<void>;
}
```

Or `SearchIndexServiceSearchIndex` 🤦 

Honestly for simplicity and to avoid polluting exports more I think it makes sense to just keep the current structure, but rename.